adding good first issue (#98)
privacy-scaling-explorations/website-v2: app/content/resources.md

    label="The Moon Math Manual"
    description="Understanding and unlocking the potential of zk-SNARKs, from beginners to experts."
    url="https://leastauthority.com/community-matters/moonmath-manual/"
  />
     <ResourceItem

>     label="Good first issues/PSE"

Reviewer: ```suggestion
    label="PSE — Good First Issues"
```

>     description="Learn by doing & contributing. Explore all the good first issues that PSE is working on."

Reviewer: ```suggestion
    description="Learn by doing & contributing. Explore all the good first issues in PSE's project catalog"
```
